packages/ckb-sdk-rpc/__tests__/ckb-rpc.test.js

```diff
@@ -117,6 +107,48 @@ describe('ckb-rpc success', () => {
   })
 })
 
+describe('send transaction', () => {
+  it('send transaction', async () => {
+    const blockHash = await rpc.getBlockHash('0')
+    const block = await rpc.getBlock(blockHash)
+    const txHash = block.transactions[0].hash
+    const tx = {
+      version: 0,
+      deps: [],
+      inputs: [
+        {
+          previousOutput: {
+            cell: {
+              txHash,
+              index: 0,
+            },
+            blockHash,
+          },
+          args: [],
+          since: '0',
+        },
+      ],
+      outputs: [
+        {
+          lock: {
+            args: [],
+            codeHash: '0x0000000000000000000000000000000000000000000000000000000000000001',
+          },
+          type: null,
+          capacity: '100000000000',
+          data: '0x',
+        },
+      ],
+      witnesses: [],
+    }
+    try {
+      await rpc.sendTransaction(tx)
```

Comment:
If not throw error, it will pass anyway ?

Reply:
updated in https://github.com/nervosnetwork/ckb-sdk-js/pull/167/commits/21c4ac413ed0f4426beae2de4a11811e7ee2f1da